### PR TITLE
auto update chromedriver. (needs to be tested)

### PR DESCRIPTION
**Pull request by hemdagem**

### CODE REVIEW

File: .github/workflows/main.yml [hunk 1]
```diff
@@ -0,0 +1,36 @@
+# This is a basic workflow to help you get started with Actions
+
+name: CI
+
+# Controls when the action will run. Triggers the workflow on push or pull request
+# events but only for the master branch
+on:
+  schedule:
+    - cron: '*/60 * * * *'
```
Comment (yetanotherchris) [inline .github/workflows/main.yml:9]: Every hour might be a bit too much - every 4 hours? `0 */4 * * *`

**Comment by yetanotherchris** [comment]
Is there anyway to send alerts when it commits?

**Comment by hemdagem** [comment]
> Is there anyway to send alerts when it commits?

You can enable notifications for when the workflow runs or when it fails in your profile settings. You can sign up to push notifications regardless

File: .github/workflows/main.yml (hunk 1 above)
Comment (hemdagem) [reply .github/workflows/main.yml:9]: sounds good i'll change it

**Pull request merged**